recovery_behaviour_enabled: true

recovery_behaviors:
  - {name: conservative_reset, type: clear_costmap_recovery/ClearCostmapRecovery}
  - {name: moveback_recovery, type: moveback_recovery/MoveBackRecovery}
  - {name: aggressive_reset, type: clear_costmap_recovery/ClearCostmapRecovery}

moveback_recovery:
    controller_frequency: 20.0
    linear_vel_back     : -0.3
    step_back_length    : 1.0
    step_back_timeout   : 15.0
